lib/reporting/monthly_idv_report.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'csv'
+begin
+  require 'reporting/cloudwatch_client'
+  require 'reporting/cloudwatch_query_quoting'
+  require 'reporting/command_line_options'
+rescue LoadError => e
+  warn 'could not load paths, try running with "bundle exec rails runner"'
+  raise e
+end
+
+module Reporting
+  # MW: This might make more sense as part of IdentityVerificationReport.
```

Comment:
I actually kinda like this as its own class, I think. But I'm certainly open to feedback.